js_modules/dagit/packages/core/src/workspace/VirtualizedJobTable.tsx

```diff
@@ -137,41 +152,99 @@ const JobRow: React.FC<{name: string; repoAddress: RepoAddress; height: number;
     return () => clearTimeout(timer);
   }, [queryJob, name]);
 
+  const {schedules, sensors} = React.useMemo(() => {
+    if (data?.pipelineOrError.__typename === 'Pipeline') {
+      const {schedules, sensors} = data.pipelineOrError;
+      return {schedules, sensors};
+    }
+    return {schedules: [], sensors: []};
+  }, [data]);
+
+  const latestRuns = React.useMemo(() => {
+    if (data?.pipelineOrError.__typename === 'Pipeline') {
+      const runs = data.pipelineOrError.runs;
+      if (runs.length) {
+        return [...runs];
+      }
+    }
+    return [];
+  }, [data]);
+
   return (
     <Row $height={height} $start={start}>
```

Comment:
Kinda random, but it might be good to add an `overflow:hidden` to this styled-component. I know back in the 2014-2015 days it could make the renderer faster because it didn't have to check to see if re-rendering one row would require re-rendering the one beneath it, but would be nice to make sure a gigantic run name etc is clipped inside the row.